mmv1/products/kms/CryptoKey.yaml

```diff
@@ -85,6 +85,34 @@ properties:
     name: 'labels'
     description: |
       Labels with user-defined metadata to apply to this resource.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'primary'
+    description: |
+      A copy of the primary CryptoKeyVersion that will be used by cryptoKeys.encrypt when this CryptoKey is given in EncryptRequest.name.
+      The CryptoKey's primary version can be updated via cryptoKeys.updatePrimaryVersion.
+      Keys with purpose ENCRYPT_DECRYPT may have a primary. For other keys, this field will be omitted.
+    output: true
+    properties:
```

Comment:
the nested properties should also have output: true to avoid confusing the generator.